Platforms/QemuQ35Pkg/Library/ConfigKnobShimLib/ConfigKnobShimLibCommon.h

```diff
@@ -0,0 +1,48 @@
+/** @file

```

Comment:
Should this library header file be under QemuQ35Pkg/Include/Library?

Reply:
I think not because it is a local header to this module, not intended for consumption outside of it.